### DISCUSSION TITLE
Cannot export .csv file

### DISCUSSION THREAD

**Discussion by starmomou** [Budibase/budibase#3431] [Help] [answered]
Hi everyone,

I'm using docker-compose to self-host Budibase. I have connected to PostgreSQL.

However, I got `JSON` file when I chose `CSV` file.
<img width="1296" alt="螢幕截圖 2021-11-18 下午2 12 47" src="https://user-images.githubusercontent.com/73826858/142363363-37357a38-aced-4031-9fb5-66cd82b0130b.png">

I opened the JSON file, but it showed `status: 400`
<img width="775" alt="螢幕截圖 2021-11-18 下午2 16 24" src="https://user-images.githubusercontent.com/73826858/142364178-7ad36b56-6dd0-4b53-8749-d9b660bcaad5.png">

I converted the `JSON` file into `CSV` file and opened it
<img width="530" alt="螢幕截圖 2021-11-18 下午2 17 12" src="https://user-images.githubusercontent.com/73826858/142364181-09bd1f48-f44a-4614-9609-048b6879f096.png">

Everything works well when I use the Budibase DB. I'm not sure what the problem is here. Thanks

**Comment by mjashanks** [comment]
Hi. I just tried this and it worked ok for me. However, I do remember that we had an issue with this, which was fixed.

What version are you running on? I am running `0.9.187`

**Reply by starmomou** [reply]
My version is also `0.9.187`.

**Reply by starmomou** [reply]
I got the same export error when I tried it with Windows 10 and Budibase `0.9.187`

**Comment by mike12345567** [comment, marked as answer]
Hi @starmomou - I think there was specifically an issue with Postgres for this, I've pushed a fix: https://github.com/Budibase/budibase/pull/3440 - will be released soon.

**Reply by starmomou** [reply]
Thank you

**Reply by starmomou** [reply]
Hi @mike12345567 I saw that the issue has been closed, but I still cannot export the correct CSV file. What may cause this happen?Docker? Postgres? 
Thanks

**Reply by mike12345567** [reply]
Hi @starmomou - the fix has not yet been released, it has been merged but we have not yet updated the images as we are still testing, this will happen soon.
